### ISSUE TITLE
Migration into Grails 3.x

### ISSUE THREAD

**Issue by magx2** [jeffellis/grails-yammer-metrics#22]
Do you have in plans to migrate into Grails 3.x?

**Comment by jeffellis** [comment]
Hi, sorry for the long delay in replying.  I'm not actively working in 
Grails these days, so I have no specific plans for upgrades at the moment.

Jeff

**Comment by magx2** [comment]
Would you be OK if I would take some parts of your code and write plugin for Grails 3? (Grails 2 and 3 are not compatible so it is easier just to start new project  and copy parts of source code)

**Comment by jeffellis** [comment]
Absolutely!  Thanks for the help.  In fact, calling it yammer-metrics is probably not appropriate anymore (since the jars are now named under dropwizard).  Feel free to reuse anything I've done and rename / take over the 3.0+ version of the plugin in the grails plugin repo.

**Comment by charu-jain89** [comment]
Hi Guys,
Where are we now as per the above discussion. Actually, I too need Grails 3.x version of this plugin.
Thanks!

**Comment by jeffellis** [comment]
Looks like someone has made one.  Check out http://grails-plugins.github.io/grails-dropwizard-metrics/

Hope that helps.

**Comment by psurolia** [comment]
This is an SNAPSHOT build "org.grails.plugins:dropwizard-metrics:1.0.0.BUILD-SNAPSHOT" , I don't think so we have release build for the same.
